### PR TITLE
Collect metrics --help flag

### PR DESCRIPTION
**Pull request by gtardif**
Signed-off-by: Guillaume Tardif <guillaume.tardif@docker.com>

**What I did**
Add --help in specific flags we catch in metrics

**Related issue**
https://docker.slack.com/archives/C0111G2RCE4/p1605266835109100

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

File: metrics/metrics.go [hunk 1]
```diff
@@ -52,18 +54,15 @@ func GetCommand(args []string) string {
 	onlyFlags := false
 	for _, arg := range args {
 		if arg == "--help" {
```
Comment (chris-crone) [inline metrics/metrics.go:56]: nit: You could make this a switch so that it's easier to read

File: metrics/metrics.go [hunk 2]
```diff
@@ -52,18 +54,15 @@ func GetCommand(args []string) string {
...
 			break
 		}
 		if isCommandFlag(arg) || (!onlyFlags && isCommand(arg)) {
-			if result == "" {
-				result = arg
-			} else {
-				result = result + " " + arg
-			}
-			if !isManagementCommand(arg) {
+			result = strings.TrimSpace(result + " " + arg)
```
Comment (chris-crone) [inline metrics/metrics.go:64]: I don't know this code well, why did you remove `if result == ""`?

Comment (gtardif) [reply metrics/metrics.go:64]: Trim spaces rather than have an if to add space only between words, saves 4 lines of code

Comment (chris-crone) [reply metrics/metrics.go:64]: I missed the addition of the TrimSpace :)

**Review by chris-crone — APPROVED** [review]
LGTM

File: metrics/metrics.go (hunk 2 above)
Comment (gtardif) [reply metrics/metrics.go:64]: This is happening in the background, to record metrics, and is followed by an http call to local Desktop, so I don't think we care. (and the args we manipulate are already trimmed)
